mmdet/models/roi_heads/test_mixins.py

```diff
@@ -207,7 +166,9 @@ async def async_test_mask(self,
             if det_bboxes.shape[0] == 0:
                 segm_result = [[] for _ in range(self.mask_head.num_classes)]
             else:
-                if rescale:
+                # TODO: check this @haian
+                if rescale and not isinstance(scale_factor,
+                                              (float, torch.Tensor)):
```

Comment:
remove float

Reply:
Would it be float?

Reply:
Would it be a Tensor?

Reply:
It is best to be able to comment on which algorithms use float and which use tensor.